### ISSUE TITLE
Reminders per day

### ISSUE THREAD

**Issue by tijlleenders** [tijlleenders/ZinZen#1999]
**Is your feature request related to a problem? Please describe.**
It's very annoying when you don't want do go all detailed in scheduling things (using duration and autoscheduler) - but just want a reminder for the day, like a birthday or some other thing that is true for the whole day.  
This is also suitable for people who feel more comfortable with a general reminder tool - not a super detailed hour-by-hour planning.

**Describe the solution you'd like**
Whenever a goal has a due date - but not a duration - show that goal in a special section of the MyTime page per day, like 'Done' and 'Trash'. Visually they will look like Tasks that come back from the scheduler, with the same options, except 'Not now'. The 'Done' option will complete the Goal and strikethrough the reminder (without disappearing).

**Describe alternatives you've considered**
None

**Additional context**
#1773 might be required for this.

**Comment by vinaybadgujar102** [comment]
blocked by #2064
